app/components/formatted_date/__snapshots__/index.test.tsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`<FormattedDate/> should default when timezone is not found 1`] = `
 <Text>
-  10:01 AM
+  6:01 PM
```

Comment:
not sure why, but this was failing on my local

Reply:
it is failing in my local too, I have a snap updated but not sent as I wasn't sure if it was only me